test/stdin/out/main.js

```diff
@@ -0,0 +1 @@
+export function working(){return"works"}
```

Comment:
Please remove this generated file and add a .gitignore rule to have git ignore it. Something like this:

`test/stdin/.gitignore`:

```
/out
```

Reply:
Gah! I complete missed this. Sorry.